README.md

```diff
@@ -349,9 +349,9 @@ refer to [android developer documentation](https://developer.android.com/trainin
 
 ### Rich push notification 
 
-In order for rich push notifications to work on your app, your app needs to be on or above version 1.2.0 of the Klaviyo SDK
+In order for rich push notifications to work on your app, your app needs to be on or above version 1.2.0-beta1 of the Klaviyo SDK
```

Comment:
```suggestion
In order for rich push notifications to work on your app, your app needs to be on or above version 1.2.0-beta of the Klaviyo SDK
```

Reply:
Thank you!

Reply:
actually I think I need to fix this to be `In order for rich push notifications to work on your app, your app needs to be on version 1.2.0-beta`

Reply:
Done.